test/relations.jl

```diff
@@ -893,24 +1143,34 @@ end
         )
         # Approximate (temperature must be computed via non-linear solve):
         @test all(
-            isapprox.(
-                internal_energy.(param_set, ts),
-                internal_energy.(param_set, ts_exact),
-                atol = atol_energy,
-            ),
-        )
-        @test all(
-            isapprox.(
-                liquid_ice_pottemp.(param_set, ts),
-                liquid_ice_pottemp.(param_set, ts_exact),
-                rtol = rtol_temperature,
+            abs.(
+                internal_energy.(param_set, ts) .-
+                internal_energy.(param_set, ts_exact)
+            ) .<= (
+                atol_energy_temperature .+
+                rtol_humidity * _eint_v0 .* getproperty.(q_pt, :tot)
             ),
         )
+        # Potential temperature comparison - use a more appropriate tolerance
+        pot_temp_errors =
+            abs.(
+                liquid_ice_pottemp.(param_set, ts) .-
+                liquid_ice_pottemp.(param_set, ts_exact)
+            )
+        # For potential temperature, use a tolerance based on the iterative nature of the calculation
+        pot_temp_tolerances = 2.25 * atol_temperature  # Larger tol to allow for condensate effects
+        failing_pot_temp = findall(pot_temp_errors .> pot_temp_tolerances)
+        # if !isempty(failing_pot_temp)
+        #     @info "Potential temperature test failures" n=length(failing_pot_temp)
+        #     @show maximum(pot_temp_errors)
+        #     @show maximum(pot_temp_tolerances)
+        # end
```

Comment:
Do you want to remove these?

Reply:
They were useful for debugging. Maybe I'll leave them for now.